src/lora/liquid_lora.py

```
@@ -0,0 +1,436 @@
+                        )
+
+                        # Update hidden state
+                        self.hidden_B[i] = new_b_row.squeeze(0)
+
+            # Use liquid-evolved B (detach for gradient flow through lora_B)
+            B_effective = self.hidden_B.detach() + self.lora_B - self.lora_B.detach()
+        else:
+            # Use standard B
+            B_effective = self.lora_B
+
+        # Compute LoRA output: (α/r) * B @ A @ x
```

Ruff (0.14.2)

Comment contains ambiguous `α` (GREEK SMALL LETTER ALPHA). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF003)